Acara/BlanKoding/BlanKoding2013.md

```diff
@@ -1,4 +1,6 @@
-# BlanKoding 2013
+---
```

Comment:
What it looks like without this lines?

Reply:
![](https://i.imgur.com/jh1l85x.png)

Reply:
Ok, I see you get rid of these lines. We'll find a way to fix the title things without adding unnecessary lines to markdown files.